### PR TITLE
feat(aha): add presence

### PR DESCRIPTION
**Pull request by vigneshpy**
## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

image
image
image
image
image
image

</details>

### CODE REVIEW

File: websites/A/aha/metadata.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+		"id": "993170607841746974",
+		"name": "பில்லி கசாப்புக்காரன்"
+	},
+	"service": "aha",
+	"description": {
+		"en": "aha is an Indian over-the-top streaming service which offers Telugu and Tamil-language content. It is owned by Arha Media & Broadcasting Private Limited, a joint venture between Geetha Arts and My Home Group"
+	},
+	"url": "www.aha.video",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/GJC1qSm.png",
```
Comment (DarkVillager) [inline websites/A/aha/metadata.json:13]: Logo's must be 512x512 as per [our guideliness](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo).

File: websites/A/aha/presence.ts [hunk 2]
```diff
@@ -0,0 +1,103 @@
+const presence = new Presence({
+		clientId: "1130920550915657860",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+const enum Assets {
+	Logo = "https://i.imgur.com/GJC1qSm.png",
```
Comment (DarkVillager) [inline websites/A/aha/presence.ts:7]: Ditto

File: websites/A/aha/presence.ts [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+						"div.player-title > span.player-label-title"
+					).textContent,
+					subtitle = document.querySelector(
+						"div.player-title > span.player-label-title__subtitle"
+					).textContent;
+				currentPresenceData.details = "Watching";
+
+				currentPresenceData.state = subtitle ? `${title} | ${subtitle}` : title;
+			} else if (pathname.includes("/account"))
+				presenceData.details = "Viewing account details";
```
Comment (DarkVillager) [inline websites/A/aha/presence.ts:98]: Please change this to the following format
```ts
switch (true) {
case pathname.startsWith("/movie"): {
				presenceData.details = `Viewing ${
					document.querySelector("h1.title").textContent
				}`;
				break;
}
// the other if statements changed to switch
}
```

**Review by DarkVillager — CHANGES_REQUESTED** [review]

**Comment by vigneshpy** [comment]
done

**Review by theusaf — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**